src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -76,10 +76,10 @@ private AdsDataProvider() {
   @DataProvider
   public static Object[][] corporatePages() {
     return new Object[][]{
-        {"wikia", "Wikia", "wka.life/_wikiaglobal//home", "CORP_TOP_LEADERBOARD"},
-        {"wikia", "About_Us", "wka.life/_wikiaglobal//article", "CORP_TOP_LEADERBOARD"},
-        {"wikia", "Alliance", "wka.life/_wikiaglobal//article", "CORP_TOP_LEADERBOARD"},
-        {"es.wikia", "Wikia", "wka.life/_corporatespanish//home", "CORP_TOP_LEADERBOARD"},
+        {"wikia", "Wikia", "wka.wikia/_wikiaglobal//home", "CORP_TOP_LEADERBOARD"},
+        {"wikia", "About_Us", "wka.wikia/_wikiaglobal//article", "CORP_TOP_LEADERBOARD"},
+        {"wikia", "Alliance", "wka.wikia/_wikiaglobal//article", "CORP_TOP_LEADERBOARD"},
+        {"es.wikia", "Wikia", "wka.wikia/_corporatespanish//home", "CORP_TOP_LEADERBOARD"},
         {"gameshub", "Games_Hub", "wka.hub/_gaming_hub//hub", "HUB_TOP_LEADERBOARD"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "gameshub" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "HUB_TOP_LEADERBOARD" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)